docs/how-to/platform-console.md

```diff
@@ -0,0 +1,120 @@
+- **Efficiency**: Quickly create new services based on pre-defined templates.
+- **Centralized Management**: Update multiple services from a single location. When you update a template, all services using that template are automatically updated.
+- **Customization**: Create and use private plugins to tailor services to your organization's specific needs.
+
+## Platform Console Templates
+
+Platform Console Templates allow you to create a single source of truth for your services' architecture. These templates can include:
+
+- Pre-configured settings
+- Standard plugins
+- Custom private plugins
+
+### Creating a Template
+
+To create a new template:
+
+1. Navigate to the Platform Console dashboard.
+2. Click on "Create Template".
+3. Provide a name for your template (e.g., "Node.js with PostgreSQL").
+4. Select the desired components (e.g., Admin UI, GraphQL API, REST API).
+5. Choose a database (e.g., PostgreSQL).
+6. Click "Create" to finalize the template.
+
+### Managing Templates
+
+Managing and updating templates is very similar to updating individual services. You have access to the same settings and options available for services, including:
+
+- Viewing all templates in the Platform Console dashboard.
+- Adding entities to the service template.
+- Adding or removing plugins, including custom plugins.
+- Renaming templates and updating their descriptions.
+- Changing the code generator version for the template.
+
+To edit a template:
+
+1. Select the template from the Platform Console dashboard.
+2. Navigate to the "Settings" tab.
+3. Make your desired changes in the "General" or "Plugins" sections.
+4. All changes are saved automatically.
+
+:::note
+When you update a template, **all services using that template will be automatically updated**, ensuring consistency across your infrastructure.
+:::
+
+### Using Templates
+
+To create a new service based on a template:
+
+1. Click on the "Add Resource" button.
+2. Select "Service from Template" from the available options.
+3. In the modal that appears, choose a specific service template.
+4. Provide a name for your new service.
+5. Click "Create" to generate the new service based on the selected template.
+
+This process ensures that the new service inherits all the pre-configured settings and plugins defined in the template.
+
+![Amplication Create Services From Template](./assets/create-service-from-template.png)
+
+## Private Plugins
+
+Private plugins allow you to create custom functionality specific to your organization's needs. These can be added to templates or directly to services.
+
+:::note
+Creating and managing private plugins requires an Enterprise plan or higher.
+:::
+
+### Creating a Private Plugin
+
+To create a private plugin:
+
+1. In the Platform Console dashboard, navigate to the "Private Plugins" section.
+2. Click on "Create Private Plugin".
+3. Follow the creation wizard to define your plugin's functionality.
```

Comment:
No wizard...
So-
Connect to your plugin repository, and add your existing private plugins to the project